WordPress/gutenberg
# Transform pasted div tags to p tags

## Description
Solves https://github.com/WordPress/gutenberg/issues/34160 

With this PR applied, content pasted from Evernote is no longer collapsed into the same paragraph:

![2021-08-18 14-18-59 2021-08-18 14_19_19](https://user-images.githubusercontent.com/205419/130043571-ccf58a5d-9dd4-4fc6-a1e9-993acba884dc.gif)

## How has this been tested?
1. Type a few paragraphs of text in Evernote
2. Copy and paste it to Gutenberg
3. Confirm the formatting was preserved

## Types of changes
Bug fix (non-breaking change which fixes an issue)

cc @ellatrix @getdave @jorgefilipecosta @draganescu @noisysocks
